src/Objectivity.AutoFixture.XUnit2.AutoMoq/SpecimenBuilder/IgnoreVirtualMembersSpecimenBuilder.cs

```diff
@@ -0,0 +1,30 @@
+namespace Objectivity.AutoFixture.XUnit2.AutoMoq.SpecimenBuilder
+{
+    using System;
+    using System.Reflection;
+    using Ploeh.AutoFixture.Kernel;
+
+    internal class IgnoreVirtualMembersSpecimenBuilder : ISpecimenBuilder
+    {
+        public object Create(object request, ISpecimenContext context)
+        {
+            if (context == null)
```

Comment:
You are right. It is remnant from code prepared for previous version of AutoFixture.